sorts/odd_even_transposition_single_threaded.py

```diff
@@ -7,11 +7,20 @@
 
 
 def OddEvenTransposition(arr):
+    """
+    >>> OddEvenTransposition([5, 4, 3, 2, 1])
+    [1, 2, 3, 4, 5]
+
+    >>> OddEvenTransposition([13, 11, 18, 0, -1])
+    [-1, 0, 11, 13, 18]
+
+    >>> OddEvenTransposition([-.1, 1.1, .1, -2.9])
+    [-2.9, -0.1, 0.1, 1.1]
+    """
     for i in range(0, len(arr)):
```

Comment:
```suggestion
    for i in range(len(arr)):
```

Reply:
This PR is mainly for doctests. Refactoring will be in other as @cclauss suggested before.